tests/Set/DoctrineORM29Set/Fixture/custom_id_generator.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping\CustomIdGenerator;
+
+class User
+{
+    /**
+     * @CustomIdGenerator(class="MyIdGenerator")
+     */
+    private $id;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping\CustomIdGenerator;
+
+class User
+{
+    #[\Doctrine\ORM\Mapping\CustomIdGenerator(class: 'MyIdGenerator')]
```

Comment:
Good question! 
The class importing can be configured by parameter in `rector.php` - see  docs https://github.com/rectorphp/rector/blob/main/docs/auto_import_names.md

It's optional parameter to avoid enforcing coding standard to people, who prefer FQN classes.

In practise most people would have this parameter enabled as you suggest :+1:

Reply:
Thanks @TomasVotruba, nice to know 👍🏼

Reply:
Wouldn't it be an idea to enable this by default? I think there are more people that import classes than ones that prefer FQCN. Those can disable `AUTO_IMPORT_NAMES`.

Reply:
@ruudk I think we had it before, but there were too many broken pieces. I guess we can re-vote :)

If you can provide me some voting resutls on Twitter or something, that would prefer the other option, we can make it default since next minor version.

Reply:
Doing a poll sounds great. I wish I had the audience/reach that you have on Twitter ;) So doing it via my Twitter won't help much.

Reply:
What about retweeting it?